### ISSUE TITLE
[Adaptive Cards-Schema Explorer]: Narrator is not giving descriptive information of the cells present under the table. 

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7907]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.Run the Narrator in scan mode using "Control+ Windows +Enter" short key.
3.Navigate to 'Schema Explorer' link using tab key and activate it using Enter key.
4.Navigate to 'Table" link using tab key which is present under "Containers" and activate using "Enter" key.
5.Navigate to the table present under the adaptive cards.
6.Observe the issue here with the narrator.

Actual Result:
Narrator is announcing like "Rows" for the "Rows" present under the table cells.

Expected Result:
Narrator should announce "Rows" like "Name column header rows 1of3".and it should give the descriptive info for all the cells present under the table.

Observation:
Same issue present for all the tables.

User Experience:
Who is dependent on-screen reader will not be able to get descriptive information of the table cells and they will be confused to understand the formation of the table.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.05.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/190989572-3fcc1689-94ff-4b24-a1e3-55a403338df7.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
Issue is not repro on the environment below, hence closing this issue.

**Test Environment:**
OS: Version 23H2 (OS Build 25915.1000)
Edge Dev: version 117.0.2045.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))
